### ISSUE TITLE
When selecting a different mod than base half-life, it freaks out and crashes (o2ds)

### ISSUE THREAD

**Issue by Hawkfang42** [masterfeizz/Xash3DS#28]
![88037438-BD0D-4EE4-B6D8-1946435C0BE4](https://user-images.githubusercontent.com/53906806/112767521-2e2e4880-8fe5-11eb-8017-6e5ab839a90a.jpeg)

**Comment by Hawkfang42** [comment]
here is the LUMA crash log
[crash_dump_00000000.zip](https://github.com/masterfeizz/Xash3DS/files/6218035/crash_dump_00000000.zip)

**Comment by Xenthio** [comment]
the mod needs to be compiled for the 3DS, so only mods that don't use custom dlls will work

**Comment by Xenthio** [comment]
this actually looks lime you forgot to put the extras.pak inside the folder of the mod you tried to launch

**Comment by Hawkfang42** [comment]
The crashing, yes, but with it, it just sends me back to the home screen, it probably is due to it using them dlls, it's a shame since it means I can't play ricochet on there.

**Comment by Xenthio** [comment]
> The crashing, yes, but with it, it just sends me back to the home screen, it probably is due to it using them dlls, it's a shame since it means I can't play ricochet on there.

yeah the libs need to be rebuilt into a format the 3ds's processor can understand

**Comment by thepwrtank18** [comment]
Happens on new 2ds too.
